docs/docsource/operation/Orbit_UI/Setting_Up_Orbit/index.rst

```diff
@@ -17,12 +17,12 @@ Orbit is configured, you can unlink it from a Zenko instance as well.
 Registering Orbit to Zenko
 --------------------------
 
-Orbit can be run as a user interface to Zenko no matter where or how
-Zenko is hosted. You can deploy Zenko:
+Orbit can be run as a user interface to Zenko no matter where or how Zenko is
+hosted. You can deploy Zenko on any host running Kubernetes 1.11.3 or
```

Comment:
I recall there may have been some issue with 1.11.x? or was that something else @tmacro

Reply:
A couple of the ease-of-use admin features I added require a newer version, but as the helm chart with blobserver hasn't been officially released yet I think we're good with `1.11.3`. The features are more for the sysadmin anyway (and are optional) and don't impact  Zenko functionality.